src/main/java/hudson/plugins/jira/JiraBuildAction.java

```diff
@@ -27,6 +27,11 @@ public JiraBuildAction(@Nonnull Set<JiraIssue> issues) {
     this.issues = new HashSet<>(issues);
   }
 
+  public JiraBuildAction(@Nonnull Run<?, ?> owner, @Nonnull Set<JiraIssue> issues) {
+    this.owner = owner;
+    this.issues = new HashSet<>(issues);
```

Comment:
so finally this should actually call `JiraBuildAction(issues)`

Reply:
Done.